docs/components/custom_components.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: Custom Components
+parent: Components
+permalink: /docs/Custom_Components
+---
+
+# Implementing a Custom Component
+
+The components we provide here may not fit your specific needs, and you'll need to write a custom component. Below is an example of how this could be done using React.
+
+## Scenario
+
+Let's say you want to use the Discovery's [Answer finding](https://medium.com/ibm-data-ai/finding-concise-answers-to-questions-in-enterprise-documents-53a865898dbd). Instead of returning _documents_, we want to get _answers_ from within those documents.
+
+## Implement a custom Answers component
+
+To make things easier, we're going to reuse the `discovery-search-app` example code -- specifically, within `src/App.js`. We'll keep using the search field, but create a custom component to show the top 3 answers.
```

Comment:
might just be me, but i wasn't sure if we meant reuse the code from src/App.js or reuse from somewhere else and add it to src/App.js. so, i would suggest:
To make things easier, we're going to reuse the 'discovery-search-app' example code. We will copy it into the src/App.js file, and then make some changes to it.

Reply:
I meant to just change `src/App.js` in place, not copy.

Reply:
just to prevent others from misunderstanding like i did, maybe:
To make things easier, we're going to edit the existing code for the 'discovery-search-app' in the src/App.js file.